### PR TITLE
NPSP: Security update for NPSP Settings Relationships Relationship Reciprocal Settings page

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata
- Classes
-- STG_PanelRelReciprocal_CTRL_TEST
# Deleted Metadata

### CODE REVIEW

File: src/classes/STG_PanelRelReciprocal_CTRL_TEST.cls-meta.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>47.0</apiVersion>
```
Comment (ptewson-sfdo) [inline src/classes/STG_PanelRelReciprocal_CTRL_TEST.cls-meta.xml:3]: This should be API v 48.0, yes?

File: src/classes/STG_PanelRelReciprocal_CTRL_TEST.cls [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+            System.assertEquals(true, hasAccess,
+                'The access check should return true if the user has access to the ' +
+                'npe4__Relationship_Lookup__c custom setting object');
+        }
+    }
+
+    @IsTest
+    static void shouldReturnFalseIfUserHasNoAccess() {
+       User adminUser = UTIL_UnitTestData_TEST.createUserWithoutInsert('Standard User');
```
Comment (ptewson-sfdo) [inline src/classes/STG_PanelRelReciprocal_CTRL_TEST.cls:54]: Should this variable be "standardUser"?

Comment (force2b) [reply src/classes/STG_PanelRelReciprocal_CTRL_TEST.cls:54]: Not to jump in on your PR, but this should be `UTIL_Profile.PROFILE_STANDARD_USER` to ensure it works in non-english org tests.

Comment (psadie-sfdo) [reply src/classes/STG_PanelRelReciprocal_CTRL_TEST.cls]: Updated!

File: src/classes/STG_PanelRelReciprocal_CTRL_TEST.cls-meta.xml (hunk 1 above)
Comment (psadie-sfdo) [reply src/classes/STG_PanelRelReciprocal_CTRL_TEST.cls-meta.xml]: Updated!

File: src/classes/STG_PanelRelReciprocal_CTRL_TEST.cls (hunk 2 above)
Comment (ptewson-sfdo) [reply src/classes/STG_PanelRelReciprocal_CTRL_TEST.cls]: Thanks @force2b ! Good catch!

**Comment by psadie-sfdo** [comment]
**lurch: attach W-041511

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000007haSOQAY" target="_blank">W-041511</a>

**Comment by psadie-sfdo** [comment]
> Looks good! Very similar to work for W-041515, yes?

Yup! Pretty much identical with the only difference being the custom setting in question.

**Pull request merged**